k8s-itlabs-operator/utils/hashing.py

```diff
@@ -0,0 +1,13 @@
+from hashlib import sha256
+from typing import Union
+
+
+def generate_hash(*args: Union[int, str]):
+    if not args:
+        raise AttributeError("No attributes to generate hash")
+
+    if not all(isinstance(i, (int, str)) for i in args):
```

Comment:
It will depend of way using this method. Initially this method created for join all required connection arguments (postgres, rabbit, sentry, keycloak and so on). Mostly of this arguments are belong for `str` and `int` types. For example:
```
postgres:
  host: postgres.local  # str
  port: 5432  # int
  database: postgres  # str
  username: vault:secret/data/postgres#USERNAME  # str
  password: vault:secret/data/postgres#PASSWORD  # str
```

No problem to modify this method, if supposed to join in common string any types. Hash in this case using only for getting unique string with fixed length.